Uplift/ViewModels/HomeGymCellViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  HomeGymCellViewModel.swift
+//  Uplift
+//
+//  Created by Caitlyn Jin on 2/24/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+extension HomeGymCell {
+
+    /// The ViewModel for a Gym Cell on the main view.
+    @MainActor
+    class ViewModel: ObservableObject {
```

Comment:
We do not need a separate ViewModel for this cell. You can use `HomeViewModel` and pass a reference of the instance to the gym cell.